Core/uWebshop.Domain/Model/OrderDTO/Order.cs

```diff
@@ -44,7 +44,7 @@ public class Order : IConvertibleToOrderInfo
 	    public DateTime? FulfillDate;
 		public PaymentInfo PaymentInfo;
 		public int PaymentProviderPrice;
-		public int? PaymentProviderOrderPercentage;
+		public decimal? PaymentProviderOrderPercentage;
```

Comment:
I think we should move on to converting that to a `decimal` but we don't know the code that is already relying on PaymentProviderOrderPercentage being a `int` * 100